services/queue/src/workerinfo.js

```diff
@@ -10,7 +10,7 @@ class WorkerInfo {
 
     // update `expires` values in postgres at this frequency; larger values give less accurate
     // expires times, but reduce database traffic.
-    this.updateFrequency = '6 hours';
+    this.updateFrequency = '30 min';
```

Comment:
It might be a good idea to make this a deployment config option, so it can be changed without making a new release or rolling back.